Raspi/Test/mock_wireless_module.py

```diff
@@ -0,0 +1,170 @@
+import time
+import random
+import argparse
+import json
+import paho.mqtt.client as mqtt
+
+
+class Sensors:
+
+    # number of decimal places for the random generated val
+    decimals = 2
+    # percent range for the rangomly generated value
+    percent_range = 0.05
+
+    def CO2(val=325):
+        # value in ppm (normal background conentration 250-400ppm)
```

Comment:
When adding documentation for a function, class or file, we add comments in a special way so that editors can recognise that this comment documents exactly what this function does, and show it as a tooltip when you hover over the function or whatever. Do this by using triple quote strings, i.e.
```python
def CO2(val=325):
    """ Value in ppm (normal background conentration 250-400ppm) """
```
This applies to all the other function documentation here. Also, typo in concentration